parsers/GB_ORK.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+"""Parser for the Orkney Islands"""
+
+import arrow
+import dateutil
+import logging
+import requests
+from bs4 import BeautifulSoup
+
+# There is a 2MW storage battery on the islands.
+# http://www.oref.co.uk/orkneys-energy/innovations-2/
+
+TZ = 'Europe/London'
+DATETIME_LINK = 'https://www.ssen.co.uk/ANM/'
+GENERATION_LINK = 'https://www.ssen.co.uk/Sse_Components/Views/Controls/FormControls/Handlers/ActiveNetworkManagementHandler.ashx?action=graph&contentId=14973&_=1537467858726'
+
+# TODO map correctly
```

Comment:
Todo?

Reply:
Not anymore I'll remove it, are you happy with the carbon intensity I've assigned to the Orkney's?

Reply:
@brunolajoie should answer that, he's more qualified than I am